pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -234,7 +235,7 @@ def update(self, request, *args, **kwargs):
             AnsibleCollectionDeprecated.objects.filter(
                 repository_version=repo_version, collection=collection
             ).delete()
-        return Response()
+        return Response(serializer.data)
```

Comment:
what motivated this change?

Reply:
the bindings client was raising a validation error because it was expecting a serialized collection - https://pulp-ansible.readthedocs.io/en/latest/restapi.html#operation/pulp_ansible_galaxy_api_v3_collections_update

Reply:
probably a changelog for this. :+1: